_posts/2022-04-19-network-upgrade-july-2022.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Monero will undergo a network upgrade on 16th July, 2022"
+summary: Announcement of the upcoming network upgrade (16th July, 2022)
+tags: [community, protocol upgrade, core, crypto]
+author: sethforprivacy
+image: /blog/assets/2022-04-19-network-upgrade-july-2022/upgrades.png
+---v
+
+<div align="center">
+    <img src="{{ page.image }}" width="350px">
+</div>
+
+The next network upgrade will take place on **16th July, 2022** (block 2,668,888). Network upgrades are technically hard-forks and require all participants in the Monero network to update their software. The new software will be released one month before the network upgrade, on the **~16th of June**.
+
+This network upgrade will introduce important new features like:
+
+- A ring-size increase from 11 to 16, increasing the base anonymity set (read: plausible deniability, or base privacy) of every transaction on the network
+- [Upgraded Bulletproofs algorithm, "Bulletproofs+"]({{ site.baseurl_root }}/2020/12/24/Bulletproofs+-in-Monero.html), which will **reduce the typical transaction size by ~5-7%** and **improve typical verification performance by ~5-7%**, making every transaction lighter and faster
+- [View tags, a brilliant way to reduce wallet sync times by 30-40%](https://localmonero.co/knowledge/view-tags-reduce-monero-sync-time)
```

Comment:
Bold

[View tags, a brilliant way to **reduce wallet sync times by 30-40%**](https://localmonero.co/knowledge/view-tags-reduce-monero-sync-time)

Reply:
Good catch, will match and be more eye catching on the important info.